Formula/c/ccache.rb

```diff
@@ -94,3 +107,31 @@ def caveats
     system "#{bin}/ccache", "-s"
   end
 end
+
+__END__
+diff --git a/cmake/FindBlake3.cmake b/cmake/FindBlake3.cmake
+index 501353c..f7d2a81 100644
+--- a/cmake/FindBlake3.cmake
++++ b/cmake/FindBlake3.cmake
+@@ -8,7 +8,7 @@ else()
+   find_library(BLAKE3_LIBRARY blake3)
+   if(BLAKE3_INCLUDE_DIR)
+     file(READ "${BLAKE3_INCLUDE_DIR}/blake3.h" _blake3_h)
+-    string(REGEX MATCH "#define _blake3_version_string \"([0-9]+).([0-9]+).*([0-9]+)\"" _ "${_blake3_h}")
++    string(REGEX MATCH "#define BLAKE3_VERSION_STRING \"([0-9]+).([0-9]+).*([0-9]+)\"" _ "${_blake3_h}")
+     set(_blake3_version_string "${CMAKE_MATCH_1}.${CMAKE_MATCH_2}.${CMAKE_MATCH_3}")
+     if(NOT "${CMAKE_MATCH_0}" STREQUAL "" AND "${_blake3_version_string}" VERSION_GREATER_EQUAL "${Blake3_FIND_VERSION}")
+       if(BLAKE3_LIBRARY)
+diff --git a/src/ccache/Hash.hpp b/src/ccache/Hash.hpp
+index 4ac7503..fc842f3 100644
+--- a/src/ccache/Hash.hpp
++++ b/src/ccache/Hash.hpp
+@@ -21,7 +21,7 @@
+ #include <ccache/util/PathString.hpp>
+ #include <ccache/util/conversion.hpp>
+ 
+-#include <blake3/blake3.h>
++#include <blake3.h>
+ #include <nonstd/span.hpp>
```

Comment:
The inline patch isn't getting used anymore is it? @carlocab

Reply:
Oops, forgot to get rid of it when I replaced it with the patch URLs

Reply:
Fixed now